inc/functions/formatting.php

```diff
@@ -102,48 +102,6 @@ function rocket_validate_js( $file ) {
 	return sanitize_text_field( \rocket_remove_url_protocol( strtok( $file, '?' ) ) );
 }
 
-/**
- * Check if the passed value is an internal URL (default domain or CDN/Multilingual).
- *
- * @since  3.3.7
- * @author Remy Perona
- * @author Grégory Viguier
- *
- * @param  string $file string to test.
- * @return bool
- */
-function rocket_is_internal_file( $file ) {
```

Comment:
You deprecated rocket_is_internal_file() function, however this function is used in:
`rocket_validate_js()` which is hooked in `rocket_sanitize_textarea_field`
```
'exclude_js'           => [ 'rocket_validate_js', 'rocket_clean_wildcards' ], // Pattern.
```

Reply:
Thank you for catching that, when I searched for it VSCode didn't return anything strangely. I undeprecated the function and updated it 👍